### ISSUE TITLE
Does not work with Bejeweled 3.

### ISSUE THREAD

**Issue by spiral6** [Codeusa/Borderless-Gaming#443]
Pretty much what it says. Attempt to test it with the Steam version and you'll see.

**Comment by Chaython** [comment]
Bejeweled 3 fullscreen behaves like borderless window. You can alt tab in/out without issue.

**Comment by spiral6** [comment]
No, it's an exclusive fullscreen, the issue lies in the fact that borderless fullscreen does not

1. force resolution changes
2. stay inactive in the background

which Bejeweled 3 does not do. It may not force resolution changes all the time, but it definitely does not stay active in the background, and instead "blacks out" the window when alt-tabbing.
